analyzers/its/expected/Ember-MM/EmberAPI-{208AA35E-C6AE-4D2D-A9DD-B6EFD19A4279}-S3900.json

```diff
@@ -145,97 +93,6 @@
 },
 {
 "id":  "S3900",
-"message":  "Refactor this method to add validation of parameter 'filename' before using it.",
-"location":  {
-"uri":  "sources\Ember-MM\EmberAPI\clsAPIFileUtils.vb",
-"region":  {
-"startLine":  113,
```

Comment:
All issues are gone except the first one on line 112
https://github.com/SonarSource/sonar-dotnet/blob/e641f52adc52e9b8743ebb3a98d0fb6582360bab/analyzers/its/sources/Ember-MM/EmberAPI/clsAPIFileUtils.vb#L111-L122